Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
I have to say that I was wondering the same and it feels like this is perfectly doable within the regular constructor (so there would be no need to have a `create` method).